doc/user/using-graalvm.md

```diff
@@ -12,9 +12,14 @@ http://www.oracle.com/technetwork/oracle-labs/program-languages/
 Inside the GraalVM is a `jre/languages/ruby` directory which has the usual
 structure of a Ruby implementation. It is recommended to add this directory to
 a Ruby manager, see [Configuring Ruby managers](ruby-managers.md) for more
-information.  
+information.
+
+You can use `gem` to install Gems as normal. We recommend that Bundler `1.14.x`
+is best to use for now.
```

Comment:
It's a bit stronger than that, only Bundler 1.14.6 is supported, otherwise there will be an error.

Reply:
Fixed.